### ISSUE TITLE
iOS 9.0 - slider is not visible

### ISSUE THREAD

**Issue by triplumix** [halkar/xamarin-range-slider#48]
I use the code below to display the range slider, but except the background color (pink, as set) I can not see any elements on that slider.

 ```
var ageSlider = new RangeSlider
			{
				MinimumValue = 17,
				MaximumValue = 70,
				LowerValue = 1,
				UpperValue = 70,
				ShowTextAboveThumbs = true,
				StepValue = 1,
				HeightRequest = 80,
				BackgroundColor = Color.Pink
			};
```

I see that the images are inside the "Resources" folder, but my app uses "Images" folder, however, I think that this does not explain why the textfields are not visible...

Any ideas on this issue?

**Comment by triplumix** [comment]
Works just fine on Android

**Comment by halkar** [comment]
1. Are you experiencing problems on iOS?
2. Is this Xamain.Forms?
3. Can you share you project XAML?
4. Try adding this code to your AppDelegate
```
using Foundation;
using UIKit;
using Xamarin.Forms;
using Xamarin.RangeSlider.Forms;

[assembly: ExportRenderer(typeof(RangeSlider), typeof(RangeSliderRenderer))]

namespace Xamarin.RangeSlider.Forms.Samples.iOS
{
```

**Comment by triplumix** [comment]
Sorry for not giving enough informations.

1. Yes, on iOS only;
2. Yes, it is XF;
3. I do not use XAML. I shared the code above and I add the slider to a relative layout. I tried setting a bigger height/width and also not setting them, same result. I believe that the slider is added to the layout and somehow visible, since I can see the pink color in background (BackgroundColor = Color.Pink)
4. I am not using it in a custom renderer, not sure if that code would change something.

I use XF 2.3.2.127, RangeSlider 0.6.10. I will try it in the next few days in a blank new page on the same project, if it is not working, I will try it in a blank new project.

**Comment by halkar** [comment]
@triplumix try code I provided.
`[assembly: ExportRenderer(typeof(RangeSlider), typeof(RangeSliderRenderer))]`

**Comment by woutermeuwis** [comment]
I think i'm stuck with the same issue on iOS 10.2
i create the component in XAML, and the code in appdelegate does not fix this

**Comment by halkar** [comment]
@triplumix @woutermeuwis I will need your XAML/code to replicate the issue.

**Comment by triplumix** [comment]
Code

```
using System.Collections.Generic;
using Xamarin.Forms;
using Xamarin.RangeSlider.Forms;

namespace MyProject
{
	public class _Test : ContentPage
	{
		public _Test()
		{
			BackgroundColor = Color.FromHex("#222");

			var layout = new RelativeLayout();

			var ageSlider = new RangeSlider
			{
				MinimumValue = 17,
				MaximumValue = 70,
				LowerValue = 1,
				UpperValue = 70,
				ShowTextAboveThumbs = true,
				StepValue = 1,
				HeightRequest = 30,
				BackgroundColor = Color.Pink
			};

			layout.Children.Add(ageSlider,
								xConstraint: Constraint.RelativeToParent((parent) => { return parent.Width * 0.05; }),
			                    yConstraint: Constraint.Constant(50),
								widthConstraint: Constraint.RelativeToParent((parent) => { return parent.Width * 0.9; }));

			Content = layout;
		}
	}
}
```

Result:

http://imgur.com/a/SL0lV

Thanks

**Comment by triplumix** [comment]
**Update:**

Works just fine on a empty new project...

Any ideas?

**Comment by triplumix** [comment]
Found the **_solution_**

You need to **_add `Xamarin.Forms.RangeSlider` package_** (NOT Xamarin.RangeSlider, this is added later as a dependency, from what I can see).

I started with an Xamarin.Forms with Android only and later added iOS to the Forms project and added every package, one by one, and by mistake I was looking at Xamarin.RangeSlider instead of `Xamarin.Forms.RangeSlider`

Thanks you for your time and sorry for wasting it :-)
I hope this will help someone who does the same mistake.

**Comment by halkar** [comment]
@triplumix didn't test it with MD. Will try it asap.

**Comment by halkar** [comment]
@triplumix TBH, I would recomend setting `IsGestureEnabled` to `false`.

**Comment by halkar** [comment]
Fixed in 0.6.16
